odl/operator/default_ops.py

```diff
@@ -877,9 +887,12 @@ def _call(self, x, out=None):
         rn(3).element([1.0, 2.0, 3.0])
         """
         if out is None:
-            return self.range.element(self.vector.copy())
+            if isinstance(self.__vector, float):
+                return self.range.element(self.__vector)
+            else:
+                return self.range.element(self.__vector.copy())
```

Comment:
A better quick fix would be to use the `copy` module from the standard library. It works with all builtin types like `float` and `complex`, and otherwise (or perhaps even there) it uses the `__copy__` method if possible. That method is implemented by `LinearSpaceVector`.